.github/workflows/functional.yml

```diff
@@ -87,7 +87,7 @@ jobs:
 
       - name: Install bitcoind
         env:
-          BITCOIND_VERSION: "28.1"
+          BITCOIND_VERSION: "29.0"
           BITCOIND_ARCH: "x86_64-linux-gnu"
           SHASUM: "07f77afd326639145b9ba9562912b2ad2ccec47b8a305bd075b4f4cb127b7ed7"
```

Comment:
You also need to add the `Install bitcoind` step (with the v29 version and proper hash) to `unit.yml` for the unit tests. They need to have `bitcoind` in their `PATH`.